packages/react-core/src/components/Form/FormSection.tsx

```diff
@@ -2,19 +2,26 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Form/form';
 import { css } from '@patternfly/react-styles';
 
-export interface FormSectionProps extends React.HTMLProps<HTMLDivElement> {
+export interface FormSectionProps extends Omit<React.HTMLProps<HTMLDivElement>, 'title'> {
   /** Content rendered inside the section */
   children?: React.ReactNode;
   /** Additional classes added to the section */
   className?: string;
+  /** Title for the section */
+  title?: React.ReactNode;
```

Comment:
@jessiehuff   are there any accessibility concerns if the title is not provided for the section?

Reply:
I think the most important thing is that the inputs have labels, but I suppose it depends on how we expect form sections to be used. If we're expecting this to be used similar to how a `<section>` element is used then it should probably have a label, as it's [required](https://www.w3.org/WAI/tutorials/page-structure/sections/#:~:text=Be%20careful%20not%20to%20overuse,should%20also%20always%20be%20labeled.) for section elements. Titles could definitely give added context if it's available, but I wonder if the idea of a form section will always have that kind of context?